sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/operations/__init__.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from ._operations import HealthApiOperations
+from ._operations import WebPubSubOperations
+
+__all__ = [
+    'HealthApiOperations',
```

Comment:
we don't need `healthApi` in the SDK although it is in the swagger, `healthApi` does not require auth. Any properties we can set in the swagger to ignore the health Api?

Reply:
Is there any problem with it? Maybe customers want to use it instead of call rest api.

Reply:
got it